integration-test/src/test/java/org/apache/iotdb/relational/it/schema/IoTDBDatabaseIT.java

```diff
@@ -617,39 +621,39 @@ public void testInformationSchema() throws SQLException {
       statement.execute(
           "create table test.test (a tag, b attribute, c int32 comment 'turbine') comment 'test'");
       statement.execute(
-          "CREATE VIEW test.view_table (tag1 STRING TAG,tag2 STRING TAG,s11 INT32 FIELD,s3 INT32 FIELD FROM s2) RESTRICT WITH (ttl=100) AS root.\"a\".**");
+          "CREATE VIEW test.view_table (tag1 STRING TAG,tag2 STRING TAG,s11 INT32 FIELD,s3 INT32 FIELD FROM s2) RESTRICT WITH (ttl=100, need_last_cache=true) AS root.\"a\".**");
```

Comment:
How does this apply to a tree view.

Reply:
The tree view will push cache in table queries, which is not restricted by tree database configurations.